libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/MapboxRouteLayerProvider.kt

```diff
@@ -205,7 +206,9 @@ internal class MapboxRouteLayerProvider(
     fun buildWayPointLayer(
         style: Style,
         originIcon: Drawable,
-        destinationIcon: Drawable
+        destinationIcon: Drawable,
+        iconOffset: List<Double> = listOf(0.0, 0.0),
+        iconAnchor: IconAnchor = IconAnchor.CENTER
```

Comment:
I think we don't need default values here since the arguments passed are coming from the options which already holds the default values. Also, default values would probably have been nice if this was a public API.